src/components/Icons/Icon/Icon.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { FiSun } from "react-icons/fi";
+// https://react-icons.github.io/react-icons/icons?name=fi
+// https://github.com/react-icons/react-icons
+
+import { Icon } from "./Icon";
+
+export default { title: "Icons", component: Icon };
+
+const args = {
+  src: <FiSun />,
```

Comment:
Hey Paul,

I don't think we need to wrap the icons in a component icon. 
Let's use it directly in the pages as: 
```
import { FiEye } from "react-icons/fi";
```
I would just set in the storybook as example of how to use and test if it displays.

Reply:
I think I understand, but in order to change colors and size etc we have to create to use the <IconContext.Provider> component provided by the lib : 
 <IconContext.Provider
      value={{
        color: props.color,
        className: props.className,
        size: props.size,
      }}
    >
      <div {...props} data-testid="icon">
        {props.src}
      </div>
    </IconContext.Provider>

Reply:
We sync in a call. All good!